app/modules/training/datastore_training.py

```diff
@@ -0,0 +1,24 @@
+import webapp2
+import logging
+import os
+from google.appengine.ext import ndb
+from google.appengine.ext.webapp import template
+from app.modules.common.kinds import Example
+
+class DatastoreTrainingHandler(webapp2.RequestHandler):
+    def get(self):
+        logging.info("DatastoreTrainingHandler")
+        
+        # get entities
+        entities_for_example_kind = Example.query().fetch(20)
```

Comment:
Rather than having 20 as a "magic number" lets make a constant `LIST_DEFAULT_LIMIT = 20` in this file and use that instead.